### ISSUE TITLE
Backend items (i.e. menu items, articles... etc) disappear for Administrator Users

### ISSUE THREAD

**Issue by 5uwebsite** [joomla/joomla-cms#23182]
### Steps to reproduce the issue
1. Login to the back-end with an user account of the default "Administrator" group. 

2. Edit any menu item or article item. Change the Access Level of the item to be "Guest".

### Expected result
The item with "Guest" Access Level setting should be seen by the "Administrator" User. 

### Actual result
The item with "Guest" Access Level setting CANNOT be seen by the "Administrator" User. It can be seen by the Super User. After the Access Level was changed to "Public", "Special" or "Registered", the item was again seen by the "Administrator" user.

### System information (as much as possible)
PHP 7.1
Joomla 3.9.1

**Comment by conseilgouz** [comment]
We already had a discussion about this last year : https://github.com/joomla/joomla-cms/issues/18542

Pascal

**Comment by 5uwebsite** [comment]
Hi Pascal,

 

Thank you for sending me the link. I totally agreed with what you said a year ago:

 

https://mrkr.io/s/5bfecb4211c3613c7b568c63/0

 

There is no way that an administrator could edit a module changed to the “Guest” level. This is a one-way operation and there is no way to come back. 

 

After reading the pull request page (https://github.com/joomla/joomla-cms/pull/18548) and the page you provided, I am still not sure what the fix is….

 

I think Administrator should see the “guest” items at the backend, regardless what level they are? After all, guest won’t have access to the backend of any Joomla site, so I think that is a reasonable suggestion?

 

Best Regards,

 

Barry Dong

Project Manager

 

5U Website - 5U means hassle free

t: 778-883-9222

f: 778-383-3857

e:    <mailto:barry@5uwebsite.com> barry@5uwebsite.com

a: 250-4140 No.3 Rd, Richmond, BC V6X 2C2

w:  <http://www.5uwebsite.com/> www.5uwebsite.com |  <http://www.温哥华网页设计.com/> www.温哥华网页设计.com

**Comment by webstractions** [comment]
@pmleconte I saw the discussion and the PR from 2017. As @5uwebsite mentioned in regards to Admin to be able to view items with Guest access was not addressed. It only covers SuperUser access to view the Guest items.

My question is where does this stand today? In my situation, I added User Login & User Registration items to a menu, set them to Guest access (User Logout was set to Registered). 

Everything appears to function correctly on the frontend, but now I am prevented from editing the menu links. I wanted to add CSS classes to the links for styling purposes, and well, I can't. I can't even trash or unpublish the links either. This is not ideal!

I do have another issue that I will be reporting shortly where the links for Guest Login/Registration appear even when the User is logged in. Refreshing the page will correct that, but this is not ideal either and not even a good workaround.
